ocpp-1-6-core/src/test/resources/FirmwareStatusNotificationRequest.json

```diff
@@ -1,24 +1,24 @@
-{

```

Comment:
it seems to be only whitespace changes, but I would avoid touching the ocpp-1-6 schemas in this PR
(this applies to the 6 schemas impacted)